boms/common-expansion/pom.xml

```diff
@@ -893,18 +878,6 @@
                 </exclusions>
             </dependency>
 
-
-            <dependency>
-                <groupId>io.smallrye.reactive</groupId>
-                <artifactId>mutiny</artifactId>
-                <version>${version.io.smallrye.smallrye-mutiny}</version>
-                <exclusions>
-                    <exclusion>
-                        <groupId>*</groupId>
-                        <artifactId>*</artifactId>
-                    </exclusion>
-                </exclusions>
-            </dependency>
             <dependency>
                 <groupId>io.smallrye.reactive</groupId>
                 <artifactId>mutiny-reactive-streams-operators</artifactId>
```

Comment:
Similarly, we need to move all artifacts from https://github.com/smallrye/smallrye-mutiny as a unit, so this one as well.

Reply:
Upon further discussion, my suggested change above is not required. It has too broad an impact relative to the fairly minor benefit of reducing the risk of misalignment in channel manifests.